tests/testsuite.src/syn_misc.at

```diff
@@ -9077,8 +9077,9 @@ AT_CHECK([$COMPILE_ONLY -fpicture-l=warning prog.cob], [0], [],
 
 AT_CLEANUP
 
+
 AT_SETUP([AREACHECK / NOAREACHECK directives])
-AT_KEYWORDS([directive])
+AT_KEYWORDS([misc directive])
 
 AT_DATA([prog.cob], [
        IDENTIFICATION  DIVISION.
```

Comment:
Please let's drop that test, along with pplex.l scanning for >> AREACHECK - this would be a GnuCOBOL only extension where we support an existing extension for the same purpose.
Sorry for not having catched that (or be clear enough about that) in the originating PR.